doc/source/features/data-inconsistency.rst

```diff
@@ -0,0 +1,37 @@
+.. _data-inconsistency:
+
+====================================================
+Data Inconsistency
+====================================================
+
+Clusters can have issues with data consistency.
+
+In one type of source and replica topology, the clients read and write to the source, and the replicas are read-only. If the clients read from the replica nodes and this information is used for writes to the source, data inconsistency may occur. This issue may also occur with the multi-master topology.
+
+In the Multi-master topology, the client can connect to any node. The effect of inconsistency is immediate. If a node has bad data, all reads propagate to bad writes. When part of the cluster has bad data, this data spreads throughout the cluster, and you cannot trust any data. Any replication error could be a sign of data inconsistency. Detecting data inconsistency is always after the fact. 
```

Comment:
You have multi-master above and Multi-master here. I think these must be unified to one style

Reply:
fixed